symfony-cmf/seo-bundle/2.0/config/packages/cmf_seo_bundle.yaml

```diff
@@ -0,0 +1,28 @@
+# Read the documentation: https://symfony.com/doc/master/cmf/bundles/seo/configuration.html
+cmf_seo:
+#    persistence:
+#        phpcr:
+#            enabled: true
+#    title: %%content_title%% | Symfony2 CMF
+#    description: The Symfony2 Content Management Framework. %%content_description%%
+#    sitemap:
+#        enabled: true
+#        defaults:
+#            default_change_frequency: always
+#            templates:
+#                html: :sitemap/index.html.twig
+#                xml: ::sitemap:index.xml.twig
+#            loaders:
+#                - _all
+#            guessers:
+#                - _all
+#            voters:
+#                - _all
+#        configurations:
+#            sitemap: ~
+#    error:
+#        exclusion_rules:
+#            - { path: ^/admin }
+#    alternate_locale:
+#        enabled: true
+#        provider_id: app.alternate_locale.provider
```

Comment:
I don't think you should put the whole config commented here:

- having commented config in your project confuses other developers having to understand why it is commented
- Flex won't replace this commented config with the new one in case you update the bundle later, so it may not even reflect the support options anymore after some time
- recipe files are not meant to replace documentation

So I suggest keeping only the most meaningful ones, and removing any option which has good defaults in the bundle and won't need to be customized immediately.